app/models/childcare_envelope.rb

```diff
@@ -1,13 +1,26 @@
 class ChildcareEnvelope < ApplicationRecord
+  ENVELOPE_STATUSES = %w[sent delivered completed declined voided].freeze
+  FAILED_ENVELOPE_STATUSES = %w[declined voided].freeze
+  COMPLETED_ENVELOPE_STATUS = 'completed'.freeze
+  IN_PROCESS_ENVELOPE_STATUSES = %w[sent delivered].freeze
```

Comment:
Would it be helpful to validate that the status is included in one of these constants (after normalizing)?

Reply:
Yes, its a good idea, when I first started with this, I was not sure on the possible statuses, I am pretty confident now that it can be locked to those 5. Good observation 👍